.github/workflows/changesets.yml

```
@@ -100,6 +100,11 @@ jobs:
           fetch-depth: 0
           token: ${{ secrets.NEO4J_TEAM_GRAPHQL_PERSONAL_ACCESS_TOKEN }}
 
+      - name: Configure git
+        run: |
+          git config --global user.name 'Neo4j Team GraphQL'
+          git config --global user.email 'team-graphql@neotechnology.com'
```

```suggestion
          git config --global user.email 'team-graphql@neotechnology.com'
```

I thought we were all on neo4j.com now?

---

Perhaps, but that should be a separate PR IMO - I think this is used in a few places, and "if it ain't broke, don't fix it!" 😆 It's just for finding the GitHub users, which still has that email address under it.